src/security/token/TKTokenCryptoEngine.m

```diff
@@ -146,6 +152,25 @@ - (TKTokenSecretKey *)_createNewKey:(Key_Level)keyLevel {
     return [TKTokenSecretKey keyWithLevel:keyLevel privateKey:privateKey publicKey:publicKey];
 }
 
+- (TKTokenSecretKey *)_createNewKey:(Key_Level)keyLevel
+                     withExpiration:(long long)expiresAtMs {
```

Comment:
no sense to have two private _createNewKey method. Use NSNumber instead of primitive long long.